auditlog/receivers.py

```diff
@@ -1,16 +1,39 @@
 import json
+from functools import wraps
 
+from django.conf import settings
+
+from auditlog.context import threadlocal
 from auditlog.diff import model_instance_diff
 from auditlog.models import LogEntry
 
 
-def log_create(sender, instance, created, **kwargs):
+def check_disable(signal_handler):
+    """
+    Decorator that passes along disabled in kwargs if any of the following is true:
+    - 'auditlog_disabled' from threadlocal is true
+    - raw = True and AUDITLOG_DISABLE_ON_RAW_SAVE is True
+    """
+
+    @wraps(signal_handler)
+    def wrapper(*args, **kwargs):
+        disable = getattr(threadlocal, "auditlog_disabled", False)
+        if not disable:
+            disable = kwargs.get("raw") and settings.AUDITLOG_DISABLE_ON_RAW_SAVE
+        kwargs.setdefault("disable", disable)
```

Comment:
_(neither supporting nor opposing the PR as a whole)_

Instead of passing `disable` flag to the signal handler, can we instead just skip calling the signal handler if we want to "disable" it?
```python
disable = getattr(threadlocal, "auditlog_disabled", False) or kwargs.get("raw") and settings.AUDITLOG_DISABLE_ON_RAW_SAVE)
if not disable:
    signal_handler(*args, **kwargs)
```

Reply:
You're absolutely right, I'll fix that right away.